shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagerSyncServiceTest.java

```diff
@@ -7,27 +7,21 @@
 
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.junit.mockito.InjectMock;
-import io.quarkus.test.kubernetes.client.WithOpenShiftTestServer;
 
 @QuarkusTest
-@WithOpenShiftTestServer
 public class ManagerSyncServiceTest {
 
     @InjectMock
     ManagedBridgeSyncServiceImpl managedBridgeSyncService;
 
     @InjectMock
-    ManagedProcessorSyncService managedProcessorSyncService;
+    ManagedProcessorSyncServiceImpl managedProcessorSyncService;
```

Comment:
Do you still need to inject the implementation now that you've refactored to an interface/implementation?

Reply:
sorry.. my mistake

Reply:
fixed now.